src/modules/AddQR/QR.js

```diff
@@ -26,17 +26,13 @@ module.exports = exports = function (options, pixels, oldPixels, callback) {
             for (var m = 0; m < width; m++) {
                 for (var n = 0; n < height; n++) {
                     if (m >= xe && n >= ye) {
-                        pixels.set(m, n, 0, qrPixels.get(m - xe, n - ye, 0))
-                        pixels.set(m, n, 1, qrPixels.get(m - xe, n - ye, 1))
-                        pixels.set(m, n, 2, qrPixels.get(m - xe, n - ye, 2))
-                        pixels.set(m, n, 3, qrPixels.get(m - xe, n - ye, 3))
+                        require('../../util/pixelSetter.js')(m,n,[qrPixels.get(m - xe, n - ye, 0),qrPixels.get(m - xe, n - ye, 1),qrPixels.get(m - xe, n - ye, 2),qrPixels.get(m - xe, n - ye, 3)],pixels);
+
                     }
 
                     else {
-                        pixels.set(m, n, 0, oldPixels.get(m, n, 0))
-                        pixels.set(m, n, 1, oldPixels.get(m, n, 1))
-                        pixels.set(m, n, 2, oldPixels.get(m, n, 2))
-                        pixels.set(m, n, 3, oldPixels.get(m, n, 3))
+                        require('../../util/pixelSetter.js')(m,n,[qrPixels.get(m, n, 0),qrPixels.get(m, n, 1),qrPixels.get(m, n, 2),qrPixels.get(m, n, 3)],pixels);
```

Comment:
I don't think you should `require` one file twice. Assign it to a variable instead.

Reply:
Yeahh, you are right I simply copied the line, and ignored that within same file it shouldn't be done twice. THANKS A LOT! 👐